Assignment2/FORGIS98_fromJSONtoJSON-LD.jsonld

```diff
@@ -0,0 +1,52 @@
+{
+  "@context" : {
+    "xsd" : "http://www.w3.org/2001/XMLSchema#",
+    "name" : "rdfs:label",
+    "middleName" : "rdfs:label",
+    "lastName" : "rdfs:label",
+    "hometown" : "rdfs:label",
+    "country" : "rdfs:label",
+    "age" : "rdfs:label",
+    "almaMater" : "rdfs:label",
+    "mothesrName" : "rdfs:label",
+    "fathesrName" : "rdfs:label",
```

Comment:
Syntactically is ok, but you have to define each attribute with a specific vocabulary attribute name because when you want to retrieve, for example your age, if you query "rdfs:label" you will not be able to distinguish what data you ask for. If you have your own vocabulary you can use it to define your attributes e.g. "name":"yourvocab:name", "age":"yourvocab:age", etc.